modules/ROOT/pages/gov-cloud-standalone.adoc

```diff
@@ -0,0 +1,45 @@
+= Standalone Mule Support in MuleSoft Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MuleSoft Government Cloud deployment permits deployment of Mule runtime engines into an on-premises environment (standalone Mules). Standalone Mules, and
+MuleSoft support for standalone Mules, are outside the scope of FedRAMP. You are solely responsible for ensuring that all data shared with Salesforce complies
```

Comment:
Can we clarify that it is the Customer's responsibility? I think that's important from the legal side

Reply:
Updated this.